modules/core_project_factory/variables.tf

```diff
@@ -258,3 +258,9 @@ variable "grant_network_role" {
   type        = bool
   default     = true
 }
+
+variable "tag_binding_values" {
+  description = "Tag values to bind the project to."
+  type        = list(string)
```

Comment:
Should this be a map(string) because you can specify the key name on `google_tags_tag_key`?

Reply:
I chose a list because we weren't passing in key information. I have addressed the main comment about why not include `google_tags_tag_key` too below.